crates/nu-command/tests/commands/assignment/append_assign.rs

```diff
@@ -78,3 +78,43 @@ fn append_assign_type_mismatch() {
 
     assert_eq!(actual.out, r#"[1,2,"a"]"#);
 }
+
+#[test]
+fn append_assign_single_element() {
+    let actual = nu!(r#"
+            mut a = ["list" "and"];
+            $a ++= "a single element";
+	    $a
+        "#);
+
+    let expected = nu!(r#"
```

Comment:
Thanks!  But I don't think an extra `nu!` is needed.  You can just use `$a == ["list", "and", "a single element"]`

It can make our tests run a little faster because we don't need to invoke nu process again